cluster/src/test/java/io/scalecube/cluster/fdetector/FailureDetectorIT.java

```diff
@@ -461,4 +459,13 @@ private void assertStatus(
       throw new RuntimeException(e);
     }
   }
+  private <T> CompletableFuture<List<T>> allOf(List<CompletableFuture<T>> futuresList) {
+    CompletableFuture<Void> allFuturesResult =
+            CompletableFuture.allOf(futuresList.toArray(new CompletableFuture[futuresList.size()]));
+    return allFuturesResult.thenApply(v ->
```

Comment:
Did not found better way how to do it :(
Need to consider better place for this method, because it can be usefull